src/containers/CreateNote/CreateNote.js

```diff
@@ -8,44 +8,58 @@ class CreateNote extends React.Component {
     super(props);
     this.state = {
       title: "",
-      noteItems: [
-        { id: uuid(), text: "default" },
-        { id: uuid(), text: "default" }
-      ]
+      noteItems: [],
+      currentFocus: null
     };
   }
+
   handleChangeTitle = event => {
-    this.setState({ title: event.target.value });
+    this.setState({ title: event.target.value, currentFocus: null });
   };
-  handleChange = event => {
+
+  handleChangeNotes = event => {
     const { noteItems } = this.state;
     let tempNoteList = [...noteItems];
-    const { id, text } = event.target;
+    const { id, value } = event.target;
```

Comment:
We were getting null before as `event.target` did not have a `text` property. This fix enables the first character you type to be saved now.